core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -237,6 +240,10 @@ private Path download(Terminal terminal, String pluginId, Path tmpDir) throws Ex
         }
 
         // fall back to plain old URL
+        if (pluginId.contains("://") == false) {
```

Comment:
Fair enough. I know it used to work in previous version but I'm fine with this implementation.
And even better it will be consistent with Kibana plugin manager which also checks that `://` exists.